src/CacheBreaker.ts

```diff
@@ -28,4 +33,4 @@ export function breakCache() {
     } else {
         fs.writeFileSync(infoFile, fileModificationTime.toString());
```

Comment:
Shouldn't that be wrapped too?

Reply:
No - it won't ever reach here if the file system is read only. `ensureDir` checks if it exists (which it wont) then tries to write so it will always error before here.

Reply:
Oh right, didn't see the return statement!